packages/theme/src/theme/_navs.scss

```diff
@@ -102,33 +102,34 @@
 
 			> a,
 			> button {
-				padding: $padding-normal;
-				letter-spacing: 0.6px;
-				line-height: 1;
-				text-transform: uppercase;
-				color: rgba(1, 1, 1, 0.6);
 				background: transparent;
 				border: none;
+				border-radius: 0;
+				color: $dark-silver;
+				letter-spacing: 0.6px;
+				line-height: 1;
+				margin-bottom: 0.5rem;
 				outline: none;
+				padding: $padding-normal;
+				text-transform: uppercase;
 
+				&:hover,
 				&:focus {
-					background: rgba(0, 0, 0, 0.06);
-				}
-
-				&:hover {
-					background: rgba(0, 0, 0, 0.03);
+					background: $concrete;
```

Comment:
no background specified in the guideline

Reply:
you miss font-weight at 600 on hover (which I don t like ..)

Reply:
I remove the background and changed the color on focus/hover.
I don't add the font-weight, it introduce an extra space on hover, pushing all following tabs. I'll tell that to ux to update the guideline